tutorials/katacoda/basic-pipeline/finish.md

```diff
@@ -0,0 +1,10 @@
+# Additional resources
+
+Now you have the core component of Tekton, Tekton Pipelines, installed on
```

Comment:
> local Kubernetes or OpenShift cluster
> Tekton CLI installed on your local machine

wording of this page should probably be updated to reflect the fact that these steps have been run in the Katacoda session, not locally.

Reply:
fixed. :)